client/reader/stream/_style.scss

```diff
@@ -240,6 +240,10 @@
 		display: block;
 	}
 
+	.reader__post-featured-video-thumbnail {
+		width: 100% !important;
```

Comment:
I'm not sure I can avoid it.
Need to combat the `!important` over here: https://github.com/Automattic/wp-calypso/pull/8640/files#diff-5fc366969c3080c52fb50121347ef0e3R226